design/FY2023/NFP-Space Sizing and HVAC.md

```diff
@@ -0,0 +1,265 @@
+
+## E-mail and Conference Call Conclusions ##
+
+n/a
+
+## Overview ##
+
+The `ZoneAirHeatBalanceAlgorithm` object has options for space heat balance.
+```
+ZoneAirHeatBalanceAlgorithm,
+    ThirdOrderBackwardDifference,  !- Algorithm
+    Yes,                      !- Do Space Heat Balance for Sizing
+    Yes;                      !- Do Space Heat Balance for Simulation
+```
+These currently result in additional output variables for Space, but nothing more.
+
+### Extend Spaces to Sizing
+When "Do Space Heat Balance for Sizing" is active, the following sizing changes will be active:
+
+1. Sizing results will be reported for every Space and Zone.
+2. Space sizing will follow the same method currently used for Zone sizing.
+3. Zone sizing will be based on coincident or non-coincident Space sizing results.
+4. Zone sizing results will be used to size air loops.
+5. Space sizing results will be used to size space-level HVAC if specified.
+
+
+### Extend Spaces to HVAC
+Inputs will be added to describe optional Space-level HVAC equipment, such as:
+
+1. Space air terminal units supplied by a single zone-level air distribution unit. e.g. A zone-level VAV damper supplying multiple space diffusers.
+2. Self-contained zone equipment, such as PTACs and fancoil units may be assigned to a Space. Maybe?
+3. Thermostats will still control a Zone, but they may be placed in a specific Space and will control that Space's temperature.
+4. Finish adding Space Name options to various features such as ZoneThermalChimney (if budget allows, otherwise restrict these to single-space zones).
+
+
+## Approach ##
+
+
+## Testing/Validation/Data Sources ##
+
+Compare Space vs Zone-level results.
+
+## Input Description ##
+Some new objects and some changes to existing objects are proposed.
+
+### ZoneControl:Thermostat:\*
+* *Change field "Zone or ZoneList Name" to "Zone or ZoneList or Space or SpaceList Name."*
+* When a thermostat is assigned to a zone, it will calculate the lumped loads to setpoint for all spaces in the zone.
+* When a thermostat is assigned to a space, it will calculate the loads to setpoint for that space.
+
+
+### SpaceHVAC:EquipmentConnections
+* *New object to connect nodes to a Space, similar to ZoneHVAC:EquipmentConnections.*
+
+```
+SpaceHVAC:EquipmentConnections,
+       \memo Specifies the HVAC equipment connections for a Space. Node names are specified for the
+       \memo Space air node and air inlet nodes.
+  A1 , \field Space Name
+       \required-field
+       \type object-list
+       \object-list SpaceNames
+  A2 , \field Space Air Node Name
+       \required-field
+       \type node
+  A3 ; \field Space Air Inlet Node or NodeList Name
+       \type node
+```
```

Comment:
Or even here is better to add the flow fraction? I assume unconditioned spaces are not allowed in a conditioned zone so each space would have this object.

Reply:
> Or even here is better to add the flow fraction? I assume unconditioned spaces are not allowed in a conditioned zone so each space would have this object.

Maybe, but this would not allow different balancing for different equipment. I'll work on expanding the NFP and adding some diagrams to consider some options here.

Also, I started with a space equipment list to allow space-level HVAC equipment, then decide that wasn't necessary, but I probably need to allow that back in, because a fan coil, say, would rarely serve more than one room (space).

As a side note, I always find the reverse maxavail flow resolution in the airloops to be rather opaque (i.e. I can't follow it in the code). So my goal is to keep this as simple and straightforward as possible.